chsdi/static/doc/examples/geoadmin_iframe.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!--[if !HTML5]>
+    <meta http-equiv="X-UA-Compatible" content="IE=9,IE=10,IE=edge,chrome=1"/>
+    <![endif]-->
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="chrome=1">
+    <meta name="viewport" content="initial-scale=1.0, user-scalable=no, width=device-width">	
+    <title>GeoAdmin Iframe example</title>
+      <style type="text/css">
+      iframe {
+        width:100%;
+        height:350px;
+      } 
+      .element {
+        background-color: white;
+        padding-left: 8px;
+        padding-bottom: 8px;
+        padding-top: 8px; 
+        width: 220px;
+        color: #555;
+        font: 11px tahoma,arial,verdana,sans-serif;
+      }
+      .title {
+        margin-top: -4px;
+        font-weight: bold;
+      }
+      </style>
+  </head>
+  <body>
+    <h1>GeoAdmin Iframe example</h1>
+    <iframe src="http://mf-geoadmin3.dev.bgdi.ch/ltgal/embed.html?X=187027.34&Y=650640.77&zoom=2&lang=fr&topic=dev&bgLayer=ch.swisstopo.pixelkarte-grau&catalogNodes=9900&layers=ch.bafu.hydroweb-messstationen_gefahren,ch.bafu.hydroweb-warnkarte_regional" name="IframeGeoadmin"></iframe>
```

Comment:
It won't work until chsdi and geoadmin are deployed. I don't think we can avoid it unless I make a template...